gnosis/cowswap
# Confirmation modal - Cow runner game

# Summary

Fixes #1764

![image](https://user-images.githubusercontent.com/622217/156810722-454a0714-d191-43f0-8d30-ade24873da29.png)


  # To Test

1. Open the [page `swap`](https://pr2528--gpswapui.review.gnosisdev.com/). Make a swap, confirm and see the confirmation modal. It should have the Cow runner game link.
